qir/qat/StaticResourceComponent/StaticResourceComponentConfiguration.cpp

```diff
@@ -29,41 +29,57 @@ namespace quantum
         config.addParameter(
             reindex_qubits_, "reindex-qubits",
             "Re-indexes statically allocated qubits with sequential ids starting from 0");
+
+        config.addParameter(
+            replace_qubit_on_reset_, "replace-qubit-on-reset", "Replaces a qubit with new qubit if the qubit is reset");
+
+        config.addParameter(
+            inline_after_id_change_, "inline-after-id-change", "Replaces a qubit with new qubit if the qubit is reset");
```

Comment:
Looks like this string accidentally got copied.

Reply:
Fixed.